containers/images/foreman/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install http://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm && \
+    yum -y install https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-repos-latest.rpm && \
+    yum -y install foreman-release-scl && \
+    yum -y install qpid-tools && \
+    yum -y install foreman tfm-rubygem-puma && \
+    yum -y install tfm-rubygem-katello tfm-rubygem-foreman_ansible tfm-rubygem-foreman_discovery tfm-rubygem-foreman_hooks && \
+    yum -y install tfm-rubygem-foreman_memcache tfm-rubygem-foreman_openscap tfm-rubygem-foreman_remote_execution tfm-rubygem-foreman_templates && \
+    yum clean all
+
+COPY container-assets/settings.yaml /etc/foreman/settings.yaml
+COPY container-assets/katello.yaml /etc/foreman/plugins/katello.yaml
+COPY container-assets/foreman_memcache.yaml /etc/foreman/plugins/foreman_memcache.yaml
+
+# Generate apipie:cache:index
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install foreman-sqlite && \
+		yum clean all
+
+COPY container-assets/build/database.yml /etc/foreman/database.yml
```

Comment:
instead of coping a temp database.yml file, you can use in memory sqlite, e.g.
```sh
DATABASE_URL=sqlite3::memory: bundle exec rake ${TASKS}
```
(abased on https://github.com/theforeman/forklift/pull/703/files#diff-e73050ae9f2baca99ce3cc574adee190R21)